docker/etc/my_init.d/10_dokku_init

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -eo pipefail
+
+log-info() {
+  declare desc="Log info formatter"
+  echo "    $*" 1>&2
+}
+
+log-fail() {
+  declare desc="Log fail formatter"
+  echo "!   $*" 1>&2
+  exit 1
+}
+
+main() {
+  if [[ ! -d /mount ]]; then
+    log-info "Creating missing /mount"
+    mkdir -p /mount
```

Comment:
any reason not to use the more standardized `/mnt` directory?

Reply:
None other than forgot about it :P

Reply:
Inside the container, I'll use `/mnt/dokku`. That seems the most appropriate, given LSB docs concerning what goes into `/mnt`.